### PR TITLE
Fix some wording

### PR DESCRIPTION
**Pull request by siaw23**
Just a few things I found worth fixing.

### CODE REVIEW

File: content/introduction/getting-started.md [hunk 1]
```diff
@@ -483,7 +483,7 @@ class Book < Hanami::Entity
 end
 ```
 
-This class will generate getters and setters for each attribute which we pass to initialize params.
+This class will generate getters and setters for each attribute which we pass to initialize `params`.
```
Comment (kaikuchn) [inline content/introduction/getting-started.md]:
```suggestion
This class will generate getters and setters for each attribute we pass to initialize `params`.
```

Comment (siaw23) [reply content/introduction/getting-started.md]: I don't understand this one. What do you mean? "which" has not been removed in this PR.

Comment (kaikuchn) [reply content/introduction/getting-started.md]: Yeah, it's the change I suggest. I think it should be removed.

**Comment by kaikuchn** [comment]
Thanks again for your contribution!

**Pull request merged**